lib/src/migrators/module.dart

```diff
@@ -174,27 +191,30 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
     // they should be forwarded or not.
     var shown = <Uri, Set<String>>{};
     var hidden = <Uri, Set<String>>{};
-    categorizeMember(Uri url, String originalName, String newName) {
+    categorizeMember(
```

Comment:
It's probably a good idea to document this. Even though it's local, the fact that it's complex enough to warrant factoring out suggests that it's probably also complex enough to warrant documenting.